pkg/3scale/amp/operator/backend_reconciler.go

```diff
@@ -38,7 +41,19 @@ func (r *BackendReconciler) Reconcile() (reconcile.Result, error) {
 	}
 
 	// Listener DC
+	backendRedisSecret := &v1.Secret{}
+	r.Client().Get(r.Context(), types.NamespacedName{
```

Comment:
Are these changes related to the current PR?

Additionally, what if the client returns error? error handling is missing?

Reply:
No these changes are from #862. I rebased off of master to pull in the TOC validation fix.